sunpy/map/mapbase.py

```diff
@@ -1785,7 +1785,17 @@ def draw_limb(self, axes=None, **kwargs):
     def draw_rectangle(self, bottom_left, *, width: u.deg = None, height: u.deg = None,
                        axes=None, top_right=None, **kwargs):
         """
-        Draw a rectangle defined in world coordinates on the plot.
+        Draw a rectangle, with corners defined in world coordinates.
+
+        This draws a rectangle that has corners at ``(bottom_left, top_right)``,
+        and that has sides parallel to the x and y axes of the plot.
+
+        If ``width`` and ``height`` are specified, they are respectively added to the
+        longitude and latitude of the ``bottom_left`` coordinate to calculate a
+        ``top_right`` coordinate. This means that ``width`` and ``height`` will only
+        correspond to the width and height of the draw rectangle if the longitude
+        and latitude coordiante axes are parallel with the x and y axes of the pixels
+        that comprise the map.
```

Comment:
@ayshih here is some text about what `width` and `height` mean. It feels a bit wordy to me, so suggestions/improvements very welcome.